app/controllers/merchant/item_orders_controller.rb

```diff
@@ -1,11 +1,15 @@
 class Merchant::ItemOrdersController < ApplicationController
   before_action :require_merchant
-  
+
   def update
     item_order = ItemOrder.find(params[:id])
-    item_order.toggle!(:status)
```

Comment:
Removed because toggle was not updating the item_order status

Reply:
You are right. If this was a boolean it would have worked better, but since it is an integer it could have been throwing things off. Good catch!